docs/language/learn-ql/introduction-to-ql.rst

```diff
@@ -1,22 +1,18 @@
 Introduction to QL 
 ==================
 
-QL is the powerful query language that underlies CodeQL, which is used to analyze code.
-Queries written with CodeQL can find errors and uncover variants of important security vulnerabilities.
-Visit `GitHub Security Lab <https://securitylab.github.com/>`__ to read about examples of vulnerabilities that we have recently found in open source projects.
-
-Before diving into code analysis with CodeQL, it can be helpful to learn about the underlying language more generally.
-
-QL is a logic programming language, so it is built up of logical formulas. QL uses common logical connectives (such as ``and``, ``or``, and ``not``), quantifiers (such as ``forall`` and ``exists``), and other important logical concepts such as predicates.
-
-QL also supports recursion and aggregates. This allows you to write complex recursive queries using simple QL syntax and directly use aggregates such as ``count``, ``sum``, and ``average``.
+Work through some simple exercises to learn about the basics of QL and CodeQL.
 
 Basic syntax
 ------------
 
 The basic syntax of QL will look familiar to anyone who has used SQL, but it is used somewhat differently.
 
-A query is defined by a **select** clause, which specifies what the result of the query should be. You can try out the examples and exercises in this topic directly in LGTM. Open the `query console on LGTM.com <https://lgtm.com/query>`__. Before you can run a query, you need to select a language and project to query (for these logic examples, any language and project will do).
+QL is a logic programming language, so it is built up of logical formulas. QL uses common logical connectives (such as ``and``, ``or``, and ``not``), quantifiers (such as ``forall`` and ``exists``), and other important logical concepts such as predicates.
+
+QL also supports recursion and aggregates. This allows you to write complex recursive queries using simple QL syntax and directly use aggregates such as ``count``, ``sum``, and ``average``.
+
```

Comment:
Suggest adding a heading for the following paragraph to avoid people skipping straight to **Simple exercises** and then wondering how to run them.